client/app/environments/dialogs/asg/asgDistribution.js

```diff
@@ -0,0 +1,41 @@
+/* Copyright (c) Trainline Limited, 2016. All rights reserved. See LICENSE.txt in the project root for license information. */
+'use strict';
+
+angular.module('EnvironmentManager.environments').factory('AsgDistributionService', function() {
```

Comment:
since that's singleton helper, I think it should be lowercase, as opposed to upper case class naming